docs/_pages/v2/guide-authentication.md

```diff
@@ -81,6 +81,29 @@ curl --user ${NETFOUNDRY_CLIENT_ID}:${NETFOUNDRY_PASSWORD} \
     --data 'grant_type=client_credentials&scope=https%3A%2F%2Fgateway.production.netfoundry.io%2F%2Fignore-scope'
 ```
 
+### NodeJS
+
+```
+const authUrl = "https://netfoundry-staging-mlvyyc.auth.us-east-1.amazoncognito.com/oauth2/token"
+const clientId = process.env.NF_CLIENT_ID
```

Comment:
This is great, @cwalkernf . Can we use the same three env vars for consistency?
* NETFOUNDRY_CLIENT_ID
* NETFOUNDRY_PASSWORD
* NETFOUNDRY_OAUTH_URL